external/report/report/_io.py

```diff
@@ -0,0 +1,28 @@
+import pathlib
+import hashlib
+import sys
+from google.cloud import storage
```

Comment:
I'm a bit skeptical about adding `google.cloud.storage` as a dependency of this package. What about having this code in `vcm.cloud`?

Reply:
Why?  It's pretty convenient to have it here...and the logic is html specific rather than general i/o. Uploading a report to a shareable link is a pretty important part of the report workflow, so makes sense to bundle it IMO. I was inspired by this feature of datapane: https://docs.datapane.com/reports/publishing-and-sharing.

GCS is a pure python package and is a google GA product, so quite stable. Adding vcm as a dependency to report would have a much larger impact on the installed footprint of report...GCS is a dep of vcm after all.